src/main/java/lotto/domain/LottoCount.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoCount {
+
+    private static final int MIN_COUNT = 0;
+    private final int count;
+
+    public LottoCount(int count) {
+        validateCount(count);
+        this.count = count;
+    }
+
+    public int get() {
+        return count;
+    }
+
+    private void validateCount(int count) {
+        if (count < MIN_COUNT) {
+            throw new IllegalArgumentException("0 이상의 숫자를 입력해야 합니다.");
+        }
+    }
+
+    public LottoCount subtract(LottoCount inputLottoCount) {
+        return new LottoCount(count - inputLottoCount.get());
```

Comment:
일반적으론 `inputLottoCount.get()`을 
`inputLottoCount.count` 또는 `inputLottoCount.getCount()`를 사용할 것 같은데 의도하신 건가요~?

Reply:
`inputLottoCount.count()` 또는 `inputLottoCount.getCount()`로 변수를 명명하지 않고, `inputLottoCount.get()`로 명명한 이유는 다음과 같은 생각에 의해서 `inputLottoCount.get()`로 명명하게 되었습니다. 
- `LottoCount` 클래스에서 getter로 꺼낼 필드가 `count` 밖에 없어서 `count()`나 `getCount()`로 변수명을 굳이 지정하지 않고 `get()`으로 변수명을 지정해도 괜찮다고 생각했습니다. 
- `LottoCount.get()`라는 변수를 보게 되면, 클래스 이름에 `LottoCount`라는 말이 들어가서 메서드에 굳이 중복적으로 `getCount()`로 메서드명을 지을 필요가 없다고 생각했었습니다. 

위와 같은 근거로 변수명을 지었는데, 혹시 문제가 될 부분이 있을까요 ... ?

Reply:
전혀 문제는 없습니다!!ㅎㅎㅎ
get()도 충분히 깔끔한 방법이라고 생각해요~!
여쭤본 이유는  제리의 의견이 궁금했어요 😄 

1. 우선 저는 위와 같은 코드에선, 일반적으로 아래와 같이 사용합니다 (자세히보면 `inputLottoCount.count()`가 아닌 `inputLottoCount.count`입니다...ㅎㅎ)
    ```suggestion
            return new LottoCount(count - inputLottoCount.count);
    ```
2. 반면 외부에서 사용할 땐 당연히 메서드로 필드에 접근해야하는데, 원하지 않아도 POJO를 만들어야 하는 상황들이 있어서 관례를 따르는 편입니다.

Reply:
새로운 적용 방식을 소개해주셔서 고마워요, 고래😄 
해주신 피드백 중에서 POJO의 의미랑 연관시켜서 이해를 하지 못했어요ㅠㅠ 
POJO의 의미에 대해서 검색해서 의미를 파악하려 했는데, 위에서 해주신 피드백이랑 연관지어서 이해하지 못했네요...

"원하지 않아도 POJO를 만들어야 하는 상황이 있어서 관례를 따르는 편입니다." 이 부분에 대해서 조금만 더 자세하게 설명 부탁드려도 될까요 ? 😂

Reply:
특정 프레임워크를 사용하다보면, 그 기능을 사용하기 위해 POJO(일반적인 getter를 포함한 객체)를 만들어야하는 상황이 있어요.
필드가 `count`이면, `getCount()`라는 메서드가 있어야 정상적으로 인식하여 동작하죠.
물론 커스텀하게 변경할 순 있어서 `get()`메서드에도 추가적인 액션을 취해주면 동일하게 작동하긴 하지만... 그 추가적인 액션이 번거롭다보니 일반적인 getter를 사용한다는 의미였습니다!

그리고 대부분의 개발자가 일반적인 getter를 사용하다보니, `get()`를 직접 들어가서 확인해봐야 할 것 같아요.
맞는 비유인진 모르겠지만, 나 혼자 `맥북`을 `맥`이라고 부르는 느낌?ㅋㅋㅋ
사람들이 맥이라고하면 노트북을 의미하는지, 데스크탑을 의미하는 지 한 번 더 물어봐야할 수도 있잖아요?ㅎㅎ
get도 마찬가지로 뭘 의미하는 지 정확히 보려면 메서드를 한 번 더 타고 들어가봐야 정확히 파악할 수 있을 것 같은 느낌이 있었어요.

**주관적인 의견이 많이 포함되어있으니 페어와 코딩하듯이 하나의 의견으로 보시길 바랍니다 😄**

Reply:
자세하게 설명해주셔서 감사합니다😄  이번에는 정말 확실하게 이해됐어요!!!
고래가 얘기해주신대로 다른 개발자들이 익숙해져있는 메서드명으로 작성할 수 있는 방법을 알 수 있어서 너무 좋았어요!!ㅎㅎㅎ